test/avocado/run-tests

```diff
@@ -257,7 +257,9 @@ def main():
         "selenium-navigate.py",
         "selenium-storage.py",
         "selenium-docker.py",
-        "selenium-sosreport.py"
+        "selenium-sosreport.py",
+        "selenium-dashboard.py",
+        "selenium-network.py"
```

Comment:
For next time: please use trailing commas, to avoid having to change existing lines when adding new ones. I. e.

            "selenium-network.py",

(Totally not important, please don't force-push just for that)